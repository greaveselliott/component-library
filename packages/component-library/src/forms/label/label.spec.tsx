import React from 'react';
import { shallow } from 'enzyme';

import Label from './label.component';

test('Renders correctly', () =>
  expect(shallow(<Label>Label text</Label>)).toMatchSnapshot());
