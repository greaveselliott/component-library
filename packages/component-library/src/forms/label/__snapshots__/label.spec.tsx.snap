// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<label
  className="label"
>
  Label text
</label>
`;
